tests/functional/spec/components/checkbox/mutually_exclusive/mutually_exclusive_number.spec.js

```diff
@@ -2,98 +2,102 @@ import NumberPage from "../../../../generated_pages/mutually_exclusive/mutually-
 import SummaryPage from "../../../../generated_pages/mutually_exclusive/mutually-exclusive-number-section-summary.page";
 
 describe("Component: Mutually Exclusive Number With Single Checkbox Override", () => {
-  beforeEach(() => {
-    browser.openQuestionnaire("test_mutually_exclusive.json");
-    browser.url("/questionnaire/mutually-exclusive-number");
```

Comment:
Same here (and other test modules), why removing the `browser.url()` from `beforeEach` and have it at the start of each test?